app/scripts/controllers/createPersistentVolumeClaim.js

```diff
@@ -24,6 +25,8 @@ angular.module('openshiftConsole')
     $scope.accessModes="ReadWriteOnce";
     $scope.claim = {};
 
+    var perpersistentVolumeClaimsVersion = APIService.getPreferredVersion('persistentvolumeclaims');
```

Comment:
perper? :)

Reply:
oh good grief, not cool.  ok, my review process sucks. lemme fix my system, sorry ☹️

Reply:
To be fair, I missed that on first review :)

Reply:
at least var name consistently wrong but not broken, heh....  Updating.